pyispyb/config.py

```diff
@@ -86,9 +86,9 @@ def get_settings() -> Settings:
 class LogConfig(BaseModel):
     """Logging configuration to be set for the server"""
 
-    LOGGER_NAME: str = "mycoolapp"
+    LOGGER_NAME: str = "py-ispyb"
     LOG_FORMAT: str = "%(levelprefix)s | %(asctime)s | %(message)s"
-    LOG_LEVEL: str = "INFO"
+    LOG_LEVEL: str = "DEBUG"
```

Comment:
What do you think about `INFO` by default and allowing to override it via env variable? I think `DEBUG` by default seems heavy.

Reply:
Personally, I'd set the db logger to `INFO` and leave the root level to `DEBUG` (ideally with the option to be override by a configuration parameter or env)
I like seeing the logs during the development but I guess it is a question of preferences of each developer.

Reply:
Personally i'd set both to info, you want the default to be correct for production. Debug is too verbose for production in my opinion and you risk things like log files becoming too big and taking down your services.

We could add a log level to the base config, then you can easily enabled debug by setting it in your local `.env` file?